src/topology/category/Top/limits.lean

```diff
@@ -125,6 +126,393 @@ instance forget_preserves_colimits : preserves_colimits (forget : Top.{u} ⥤ Ty
     by exactI preserves_colimit_of_preserves_colimit_cocone
       (colimit_cocone_is_colimit F) (types.colimit_cocone_is_colimit (F ⋙ forget)) } }
 
+lemma limit_induced (F : J ⥤ Top.{u}) :
+  (limit F).str = ⨅ j, (F.obj j).str.induced (limit.π F j) :=
```

Comment:
I'm not sure whether we have an established way to discuss properties about the topological structure of an object `X : Top`. @semorrison @PatrickMassot what do you think?

In any case, we should be consistent about using `X.str` (as yoou do here) or `X.topological_space` (as appears below).

Reply:
I've changed everything to `X.topological_space` for now.